faq_bot/README.md

```diff
@@ -0,0 +1,27 @@
+## Description
+
+Example FAQ bot built on `dff`. Uses telegram as an interface.
+
+## Testing the bot
+```commandline
+docker build -t telebot-test --target test .
+```
+
+## Running the bot
+
+### With docker
+
+```commandline
+docker build -t telebot-prod --target prod .
+docker run --env-file .env telebot-prod
+```
+
+### With docker-compose
+
+```commandline
+docker-compose up
+```
+
```

Comment:
Add info about updating of .env. docker-compose should be first
(You can use ChatGPT for more detailed description)

Reply:
https://github.com/deeppavlov/dialog_flow_demo/pull/1/commits/7b9d6521717f1d9073a1245b8e163d1b67c41618